internal/k8s/watch.go

```diff
@@ -294,6 +294,10 @@ func (kCli K8sClient) watchableGroupVersionResources(ctx context.Context) ([]sch
 		}
 	}
 
+	if len(ret) == 0 {
```

Comment:
didn't we say that it made more sense to check the number of things returned in the caller of this func? though meh i don't have strong feels

Reply:
Yeah, the problem is the thing that calls this returns a channel, and you can't exactly check the length of a channel. I could move it up one level though.